### DISCUSSION TITLE
Help With Playlist

### DISCUSSION THREAD

**Discussion by treboa** [TT-ReBORN/Georgia-ReBORN#155] [Q&A] [answered]
Hi TT (and any readers)
 Firstly, many thanks for your wonderful Georgia-ReBorn theme. I am new (read no idea!) to foobar2000 having changed from many years of MM (4.1.31) after problems with FLAC streaming. I installed foobar as portable, added your theme and the upnp plugin,  spent several hours trying to familiarise myself with settings and all is good and working fine. I managed to make a _very_ minor alteration to the library tree track row, ie: 01. track title to 01 - track title which now matches my very large music library but I cannot work out how to edit or script the playlist track view and, perhaps, add more info, say composer. I have searched and I found the custom box in options but, forgive my ignorance, could you possibly show me exactly what to do, perhaps with an example. I have found 4 config folders (2 are 3.0-RC2) in my foobar directory but lack the skill to proceed (so far!). Again, I realise this is very basic stuff but I would really appreciate your valuable help.
treboa :)

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @treboa,

you can do that via top menu `Options` > `Playlist` > `Album header` > `Customize header info`. 
This is a shortcut to write your custom pattern in your `georgia-reborn-config.jsonc` file in section:
`"settings":`
`"playlistCustomHeaderInfo": "", // You can use your own custom pattern for the playlist header info`
which you can also write it directly in your config file.

You can also do the same for title row -> `Options` > `Playlist` > `Track row `> `Customize track row`
`"settings":`
`"playlistCustomTitle": "", // You can use your own custom title pattern for the playlist row`

_Note: If both are blank again, it will use the default patterns._

**Example of pattern usage for playlist row:**
```
"playlistCustomTitle": "[%artist% - %album% -  ]%title%[ '('%original artist%' cover)']", // You can use your own custom title pattern for the playlist row
"playlistCustomTitleNoHeader": "[%artist% - %album% -  ]%title%[ '('%original artist%' cover)']", // You can use your own custom title pattern for the playlist row - when playlist header is not being displayed
```

-TT

**Reply by treboa** [reply]
You are very kind, TT. My education continues! I’ll try to follow your advice.

best wishes,
treboa
